akash/totalPrice.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Task 5 : Calculating total price of the purchased products
+ * 
+ * @author Akaash
+ * @version 1.0
+ * @since 05/03/2020
+ */
+const readline = require('readline')
+
+const read = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+})
+
+let productList = []
+
+let retry = function(){
```

Comment:
use const instead of let.

Reply:
replaced const instead of let and pr raised